content/en/blog/2022/exponential-histograms/index.md

```diff
@@ -1,16 +1,15 @@
 ---
 title: "Exponential Histograms: Better Data, Zero Configuration"
 linkTitle: Exponential Histograms
-date: 2022-08-22
+date: 2022-08-23
```

Comment:
I think we should update this to 24: having multiple blog posts with the same date, leads to some strange behaviour, see https://github.com/open-telemetry/opentelemetry.io/pull/1659

Reply:
Agreed, let's make this today's date.

Other than that, is the ready for your approval @svrnm?
If so, after you approve, I'll update the date and then merge.

Reply:
Ha, forgot what day it was. Fixed.